lib/msal-browser/src/broker/BrokerAuthResult.ts

```diff
@@ -0,0 +1,30 @@
+import { BrokerMessage } from "./BrokerMessage";
+import { BrokerAuthenticationResult } from "@azure/msal-common";
```

Comment:
Nit: We have two files named `BrokerAuthResult` and `BrokerAuthenticationResult`, is the difference `common` and `browser`?

Reply:
We should probably rename `BrokerAuthResult` to `BrokerAuthResponse`